content/docs/command-reference/install.md

```diff
@@ -75,11 +75,11 @@ directory:
 To disable them, you need to **remove** or **edit** those files (i.e.
 `rm .git/hooks/post-checkout`, `vim .git/hooks/pre-commit`).
 
-## Using Pre-Commit tool
+## Using the **pre-commit** tool
```

Comment:
make it inline code `pre-commit`?

Reply:
Since it's the official name of the tool I just wanted to bold it but I see that it has no effect in the rendering since the header is already bold:

![image](https://user-images.githubusercontent.com/1477535/77510282-02a5da80-6e34-11ea-96be-0f6f1c75c5fb.png)

Will change...

Reply:
Done in 9aea2690